### ISSUE TITLE
PanelMenu: Invalid ARIA attribute: aria-controls

### ISSUE THREAD

**Issue by markuczy** [primefaces/primeng#16447]
### Describe the bug

When using PanelMenu, if menuItem does not have children, element with id set as `aria-controls` does not exist.

Only when menuItem is menuItemGroup, id is added to a div https://github.com/primefaces/primeng/blob/master/src/app/components/panelmenu/panelmenu.ts#L810.

You can observe the issue using Axe DevTools on PanelMenu documentation page.
![image](https://github.com/user-attachments/assets/eb486505-8f62-49ee-8e00-58faabae1657)

### Environment

Ubuntu 22.04, WSL 2, Windows 11 PRO

### Angular version

17

### PrimeNG version

17

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20.12.2

### Expected behavior

Expected PanelMenu to not cause accessibility issue
